addons/pos_self_order/static/src/app/pages/order_history_page/order_history_page.js

```diff
@@ -4,16 +4,18 @@ import { Component, useState } from "@odoo/owl";
 import { useSelfOrder } from "@pos_self_order/app/self_order_service";
 import { _t } from "@web/core/l10n/translation";
 import { useService } from "@web/core/utils/hooks";
-
+import { OrderReceipt } from "@point_of_sale/app/screens/receipt_screen/receipt/order_receipt";
 export class OrdersHistoryPage extends Component {
     static template = "pos_self_order.OrdersHistoryPage";
+    static components = { OrderReceipt };
```

Comment:
`OrderReceipt` seems to be unused in this component.

Reply:
indeed